src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.tsx

```diff
@@ -21,39 +21,40 @@ export interface KeyStatConfig {
   value: string;
 }
 
-const KeyStatTile = ({
-  dataBlockRequest,
-  summary,
-  dataBlockResponse: response,
-  children,
-  handleApiErrors,
-}: KeyStatProps) => {
+const KeyStatTile = ({ dataBlockRequest, summary, children }: KeyStatProps) => {
+  const { withoutErrorHandling } = useErrorControl();
   const [dataBlockResponse, setDataBlockResponse] = useState<
     DataBlockResponse | undefined
-  >(response);
-
-  const [config, setConfig] = useState<KeyStatConfig | undefined>();
+  >();
 
   useEffect(() => {
-    if (!dataBlockResponse) {
-      DataBlockService.getDataBlockForSubject({
-        ...dataBlockRequest,
-        includeGeoJson: false,
-      })
-        .then(setDataBlockResponse)
-        .catch(handleApiErrors);
-    } else {
+    withoutErrorHandling(async () => {
```

Comment:
Don't need `withoutErrorHandling`. The previous behaviour was to `handleApiErrors` and that is the default for every service method now.

You only need `withoutErrorHandling` when you actually want to do something custom if an error is thrown, e.g. show some custom fallback UI.

Reply:
I think that Mark has resolved this now @ntsim :+1: